test/src/code/code_test.dart

```diff
@@ -11,7 +12,7 @@ final _language = java;
 const _text = '''
 1 Lorem ipsum dolor sit amet,
 2 consectetur adipiscing elit,
-3 sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+3 sed do eiusmod tempor incididunt ut labore et dolore magna.
```

Comment:
can't "aliqua" be added to the line #73 instead?

Reply:
`+` Good idea, less changes, thanks.